mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -80,7 +80,11 @@ CMomentumStickybombLauncher::~CMomentumStickybombLauncher()
             const auto pStickybomb = m_Stickybombs[i];
             if (pStickybomb)
             {
+#ifdef CLIENT_DLL
                 pStickybomb->Destroy(true);
+#else
+                pStickybomb->Fizzle();
+#endif
```

Comment:
Not sure why this ifdef block is needed or how this even compiles -- wasn't `Destroy(bool)` removed?

Reply:
`Fizzle` is server only and client was calling a `Destroy` function earlier, which isn't the same one:
![image](https://user-images.githubusercontent.com/9014762/97141396-e21c6680-171b-11eb-9616-0ac99e499475.png)

Reply:
Actually looks like [that function](https://github.com/momentum-mod/game/blob/e3efc48eb58cf56af61b2fef8d0bf7e3bf754680/mp/src/game/shared/baseprojectile.h#L57) isn't doing anything anyways. I'll remove that I suppose.